src/charts.js

```diff
@@ -731,16 +767,14 @@
     };
 
     var clearZoomLevelButtonActiveStates = function(){
-      $('.time-button').removeClass('active');
+      d3.select('.time-button').classed('active', false);
     };
 
     var $toolbar = null;
     if ( interfaceSettings.toolbar.enabled ){
-
-      $toolbar = $('<div></div>');
-      $toolbar.addClass('omh-chart-toolbar');
-      $toolbar.attr('unselectable', 'on');
-      $element.append( $toolbar );
+      $toolbar = $element.append("div")
```

Comment:
Since $toolbar is returned by getComponents(), this is technically a breaking change for anyone expecting $toolbar to be a jQuery object.